### ISSUE TITLE
Deprecated lifecycle method warnings with React 16.9

### ISSUE THREAD

**Issue by nikhiln** [Availity/availity-reactstrap-validation#155]
The following warnings are shown upon upgrading to React 16.9.0:

```
Warning: componentWillMount has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

* Move code with side effects to componentDidMount, and set initial state in the constructor.
* Rename componentWillMount to UNSAFE_componentWillMount to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: AvForm, AvInput
```

And

```
Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

* Move data fetching code or side effects to componentDidUpdate.
* If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://fb.me/react-derived-state
* Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: AvInput
```

Since react v16.9.0, it started to display warnings for deprecated lifecycles, is there any PR already open for this issue?

**Comment by GoPro16** [comment]
There is no PR open for this yet but we are migrating to the latest version with hooks. If you want to fix this warning for older versions feel free to submit a PR.

**Comment by mrsarm** [comment]
This is the PR that solves the issue https://github.com/Availity/availity-reactstrap-validation/pull/159 , but because it can take a while the patch to be merged, you can just use my forked version: https://www.npmjs.com/package/availity-reactstrap-validation-safe
